llvm/include/llvm/Passes/CodeGenPassBuilder.h

```diff
@@ -956,6 +958,8 @@ Error CodeGenPassBuilder<Derived, TargetMachineT>::addMachinePasses(
   if (getOptLevel() != CodeGenOptLevel::None)
     derived().addBlockPlacement(addPass);
 
+  derived().addBlockShuffling(addPass);
+
```

Comment:
`if (isFuzzed(l1i ...)`?

Reply:
Hmmm, yeah. It's better to this that way you show. Currently, it's checked right inside the pass and if it is not required to go to this fuzz, it will just return from there without any modification